scripts/evaluate_release_tag.py

```diff
@@ -41,7 +40,7 @@ def git_plain_tag_versions(versions: List[Version]) -> List[Version]:
 
 
 def filter_non_alpha_releases(tags: List[Version]) -> List[Version]:
-    return [tag for tag in tags if not tag.is_alpha]
+    return [tag for tag in tags if tag.is_alpha is False]
```

Comment:
During the merge, a conflict happened, @radovanZRasa  please check I resolved it correctly.